readthedocs/projects/models.py

```diff
@@ -280,11 +282,11 @@ def save(self, *args, **kwargs):
             log.error('failed to update static metadata', exc_info=True)
         try:
             branch = self.default_branch or self.vcs_repo().fallback_branch
-            if not self.versions.filter(slug='latest').exists():
+            if not self.versions.filter(slug=LATEST).exists():
                 self.versions.create(
-                    slug='latest', verbose_name='latest', machine=True, type='branch', active=True, identifier=branch)
-            # if not self.versions.filter(slug='stable').exists():
-            #     self.versions.create(slug='stable', verbose_name='stable', type='branch', active=True, identifier=branch)
+                    slug=LATEST, verbose_name=LATEST_VERBOSE_NAME, machine=True, type='branch', active=True, identifier=branch)
+            # if not self.versions.filter(slug=STABLE).exists():
+            #     self.versions.create_stable(type='branch', identifier=branch)
```

Comment:
It's not in that case. But here: https://github.com/rtfd/readthedocs.org/pull/1354/files#diff-c31d0a7752163caf020c33fb85ba7a96R124